apps/api/src/hyp3_api/routes.py

```diff
@@ -162,7 +163,10 @@ def __init__(self, spec):
 
     def post(self):
         body = request.get_json()
-        return jsonify(handlers.post_subscriptions(body, g.user))
+        validate_only = deepcopy(body.get('validate_only'))
+        if validate_only is not None:
+            del body['validate_only']
```

Comment:
this kind of logic is enough for me to want to move the `validate_only` key outside of the subscription object, e.g:
```
{
  "subscription": {
    "search_parameters": {...},
    "job_specification": {...}
  }
  "validate_only': false
}
```
but that's cumbersome for the 99.9% of cases where you're not going to bother specifying `validate_only`

Reply:
otherwise `validate_only = body.pop('validate_only', False)` might be equivalent and more concise